python/mxnet/gluon/model_zoo/vision/mobilenet.py

```diff
@@ -74,13 +124,71 @@ def hybrid_forward(self, F, x):
         x = self.output(x)
         return x
 
+
+class MobileNetV2(nn.HybridBlock):
+    r"""MobileNetV2 model from the
+    `"Inverted Residuals and Linear Bottlenecks:
+      Mobile Networks for Classification, Detection and Segmentation"
+    <https://arxiv.org/abs/1801.04381>`_ paper.
+
+    Parameters
+    ----------
+    multiplier : float, default 1.0
+        The width multiplier for controling the model size. The actual number of channels
+        is equal to the original channel size multiplied by this multiplier.
+    classes : int, default 1000
+        Number of classes for the output layer.
+    """
+
+    def __init__(self, multiplier=1.0, classes=1000, **kwargs):
+        super(MobileNetV2, self).__init__(**kwargs)
+        with self.name_scope():
+            self.features = nn.HybridSequential(prefix='features_')
+            with self.features.name_scope():
+                _add_conv(self.features, int(32 * multiplier), kernel=3,
+                          stride=2, pad=1, active=False)
```

Comment:
no `relu` for the first conv?

Reply:
The MobileNetV2 paper not mentioned relu after first conv2d.
Also the channels of first conv2d is 32, while the channels of first bottleneck is 16, this doesn't sense.
I have tested 3 different model architectures:
- with relu in first conv2d
- without relu in first conv2d
- without relu and use 16 instead of 32 as output channels in first conv2d

The result is a bit confusing, since performance among those architectures looks similar.
Here is the result:
![test](https://user-images.githubusercontent.com/5899962/36290680-8fd83514-1301-11e8-8de6-f69b2d16e2bc.png)

The dataset contains 100 classes, 19W pics, all gray-scale image. This dataset maybe too small. But I don't have enough resource to test on coco or imagenet.